src/rest_api/classes/gene/widgets/phenotype.clj

```diff
@@ -421,34 +437,30 @@
                                                       tg-id)]
                  evidence)))}}))
 
-(defn drives-overexpression [gene]
-  (let [db (d/entity-db gene)
-        tg-ids (d/q '[:find [?tg ...]
-                      :in $ ?gene
-                      :where
-                      [?cbg :construct.driven-by-gene/gene ?gene]
-                      [?cons :construct/driven-by-gene ?cbg]
-                      [?cons :construct/transgene-construct ?tg]]
-                    db (:db/id gene))
-        tg-evidence (partial transgene-evidence-for-phenotype db tg-ids)
-        phenotype
-        (->> tg-ids
-             (map (fn [tg]
-                    (->> (d/q '[:find [?pheno ...]
-                                :in $ ?tg
-                                :where
-                                [?tg :transgene/phenotype ?ph]
-                                [?ph
-                                 :transgene.phenotype/phenotype
-                                 ?pheno]]
-                              db tg)
-                         (map tg-evidence)
-                         (into {}))))
-             (into {}))]
-    {:data (if ((comp not empty?) phenotype)
-             {:Phenotype phenotype})
-     :description (str "phenotypes due to overexpression under "
-                       "the promoter of this gene")}))
+(defn- phenotype-table-entity-overexpressed
```

Comment:
Prefer shorter function names (especially when they're private) e.g `overexpressed-tbl` would be fine?)